docs/docs/themes/introduction.md

```diff
@@ -1,51 +1,31 @@
 ---
-title: Gatsby Themes
+title: What are Gatsby themes?
```

Comment:
```suggestion
title: What Are Gatsby Themes?
```
Using title case per the [Gatsby Style Guide](https://www.gatsbyjs.org/contributing/gatsby-style-guide/#format-titles-and-headers)